reporting/pom.xml

```diff
@@ -118,6 +118,14 @@
             <groupId>net.spals.appbuilder.plugins</groupId>
             <artifactId>spals-appbuilder-app-finatra-test</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.spals.appbuilder.plugins</groupId>
+            <artifactId>spals-appbuilder-app-grpc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>net.spals.appbuilder.plugins</groupId>
+            <artifactId>spals-appbuilder-app-grpc-test</artifactId>
+        </dependency>
```

Comment:
Note that the `app-examples` module is skipped here as that doesn't need to be a part of the code coverage report.